src/SAML2/Attribute/AttributeSet.php

```diff
@@ -19,24 +19,17 @@
 namespace Surfnet\SamlBundle\SAML2\Attribute;
 
 use ArrayIterator;
-use Countable;
-use IteratorAggregate;
 use SAML2_Assertion;
 use Surfnet\SamlBundle\Exception\RuntimeException;
 use Surfnet\SamlBundle\SAML2\Attribute\Filter\AttributeFilter;
 
-final class AttributeSet implements IteratorAggregate, Countable
+class AttributeSet implements AttributeSetFactory, AttributeSetInterface
```

Comment:
I was in doubt about changing the name. I guess the external API would not change if I were to call it "DefaultAttributeSet" or something, so it would technically *not* be a BC break.